test/test_sort_and_select.py

```diff
@@ -751,6 +751,115 @@ def test_kthvalue_scalar(self, device, dtype):
         self.assertEqual(res[0], ref[0].squeeze())
         self.assertEqual(res[1], ref[1].squeeze())
 
+    @dtypes(*set(torch.testing.integral_types()))
+    @onlyCPU
+    def test_isin(self, device, dtype):
```

Comment:
Make sure to add test cases for error checks, too. Another option that may simplify the testing is to define a set of inputs and then compare the output of torch.isin() with np.isin().

Reply:
There aren't any error cases per se (the function doesn't throw), but this did get me thinking about different dtypes for the two inputs. Added a test that verifies all combinations of integral types.

Since `test_elements` isn't an input to TensorIterator, type promotion isn't done automatically. Manual type promotion is required to pass the tests, so I added that as well.

I also replaced the `_isin_slow()` implementation with `np.isin()` to validate against the reference implementation.

Reply:
You'll also want to check that the tensors are on the same device, for example.

Reply:
Right now, only CPU is supported